src/utils/types/zetkin.ts

```diff
@@ -212,6 +213,7 @@ export interface ZetkinSession {
   created: string;
   level: number;
   user: ZetkinUser;
+  factors: string[];
```

Comment:
I would type this more strictly than this. The possible choices are `email_password`, `phone_otp` and `api_key`.

```suggestion
  factors: ('email_password','phone_otp','api_key')[];
```